mitosheet/src/mito/Mito.tsx

```diff
@@ -334,54 +332,6 @@ export const Mito = (props: MitoProps): JSX.Element => {
         previousNumSheetsRef.current = sheetDataArray.length;
     }, [sheetDataArray])
 
-    const previousNumGraphsRef = useRef<number>(Object.keys(analysisData.graphDataDict || {}).length)
-    const previousGraphIndex = useRef<number>(uiState.selectedGraphID !== undefined ?
-        Object.keys(analysisData.graphDataDict  || {}).indexOf(uiState.selectedGraphID) : -1)
-
-    // When we switch graphID's make sure that we keep the previousGraphIndex up to date
-    useEffect(() => {
-        previousGraphIndex.current = uiState.selectedGraphID !== undefined ?
-            Object.keys(analysisData.graphDataDict || {}).indexOf(uiState.selectedGraphID) : -1
-    }, [uiState.selectedGraphID])
-
-    // Update the selected sheet tab when the number of graphs change. 
-    useEffect(() => {
-        const graphIDs = Object.keys(analysisData.graphDataDict || {})
```

Comment:
Do we still want this effect? It would be different behavior -- but when there's a graph, switching to it?